api/v1alpha1/annotations.go

```diff
@@ -0,0 +1,7 @@
+package v1alpha1
+
+const (
+	AnnotationKeyRefresh      = "kargo.akuity.io/refresh"
+	AnnotationKeyRBACGroups   = "kargo.akuity.io/rbac.groups"
+	AnnotationKeyRBACSubjects = "kargo.akuity.io/rbac.sub"
+)
```

Comment:
I really like the placement of these here. There are probably other labels and annotations we could move to here in a separate PR.